src/simplewallet/simplewallet.h

```diff
@@ -125,7 +125,8 @@ namespace cryptonote
     bool help(const std::vector<std::string> &args = std::vector<std::string>());
     bool start_mining(const std::vector<std::string> &args);
     bool stop_mining(const std::vector<std::string> &args);
-    bool save_bc(const std::vector<std::string>& args);
+    bool set_node(const std::vector<std::string> &args);
+    bool save_bc(const std::vector<std::string> &args);
```

Comment:
Please don't modify lines around your changes.

Reply:
The `&` was on the other side, so I thought I'd add that cosmetic change in, should I revert this?

Reply:
Generally, I frown upon these, as it creates more changes for no reason. However, when this is within an existing hunk, I'm usually fine to restyle.  That kind of change seems just arbitrary though, it's not like you're fixing bad indentation. If you really want to keep this in, I'll be OK with it, but in general, please avoid.

Reply:
Ok, I'll leave it fixed then.